packages/help-center/src/components/help-center-search-results.tsx

```diff
@@ -252,13 +252,9 @@ function HelpSearchResults( {
 				section: sectionName,
 			} );
 
-			// push state only if it's internal link.
-			if ( ! /^http/.test( link ) ) {
```

Comment:
@arcangelini In the links, we have [external](https://github.com/Automattic/wp-calypso/pull/95284/files#diff-f8188ad4da9424adbc6f6686433278ab2461f4508f2620c3d6ce55ebaf631d89L59) links that in this case would not work ( [this](https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/contextual-help/admin-sections.ts#L388) for example ). Why did we have to do this here?

Maybe I can, instead of removing the check, make an exception for the admin sections that are not internal?